include/string.h

```diff
@@ -0,0 +1,29 @@
+///////////////////////////////////////////////////////////////////////////////
+//// Licensed Materials - Property of IBM
+//// ZOSLIB
+//// (C) Copyright IBM Corp. 2022. All Rights Reserved.
+//// US Government Users Restricted Rights - Use, duplication
+//// or disclosure restricted by GSA ADP Schedule Contract with IBM Corp.
+/////////////////////////////////////////////////////////////////////////////////
+//
+#ifndef _ZOS_STRING_H
+#define _ZOS_STRING_H
+
+#include_next <string.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+size_t strnlen(const char *, size_t );
```

Comment:
We'll need to remove https://github.com/ibmruntimes/zoslib/blob/48231f1268a3c522732f90966b6d144ac45242b6/include/zos-base.h#L432 and it's definition so it doesn't conflict